momepy/utils.py

```diff
@@ -38,18 +38,13 @@ def unique_id(objects):
 
 def _angle(a, b, c):
     """
-    Measure angle between a-b, b-c. In radians.
+    Measure angle between a-b, b-c. In degrees.
     Helper for gdf_to_nx.
+    Adapted from cityseer's implementation.
     """
-    ba = [aa - bb for aa, bb in zip(a, b)]
-    bc = [cc - bb for cc, bb in zip(c, b)]
-    nba = math.sqrt(sum((x ** 2.0 for x in ba)))
-    ba = [x / nba for x in ba]
-    nbc = math.sqrt(sum((x ** 2.0 for x in bc)))
-    bc = [x / nbc for x in bc]
-    scal = sum((aa * bb for aa, bb in zip(ba, bc)))
-    angle = math.acos(round(scal, 10))
-    return angle
+    a1 = math.degrees(math.atan2(b[1] - a[1], b[0] - a[0]))
```

Comment:
Is there a specific reason for using `math` instead of `numpy` for these calculations?

Reply:
Yes, it is way faster for scalars. numpy would make sense if we used arrays here, which we don't (we may refactor it, though)

```py
%%timeit
np.rad2deg(np.arctan2(1, 0))

1.26 µs ± 47.4 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

%%timeit
math.degrees(math.atan2(1, 0))

100 ns ± 3.83 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
```